tests/operator/operator.go

```diff
@@ -2407,10 +2407,21 @@ spec:
 			}
 		})
 
-		It("[test_id:3153][QUARANTINE] Ensure infra can handle dynamically detecting DataVolume Support", decorators.Quarantine, func() {
+		It("[test_id:3153] Ensure infra can handle dynamically detecting DataVolume Support", decorators.Quarantine, func() {
```

Comment:
@akalenyu - I think you need to drop the quarantine decorator too - the test is still being skipped.

Reply:
Damn right thx